internal/common/config/config.go

```diff
@@ -47,7 +47,11 @@ type ForagingConfig struct {
 	MaxDeerPopulation     uint    // Maximimum possible deer population. Reserved for post-MVP functionality
 	DeerGrowthCoefficient float64 // Scaling parameter used in the population model. Larger coeff => deer pop. regenerates faster
 
-	// TODO: add other pertinent params here (for fishing etc)
+	// Fih hunting
+	MaxFishPerHunt            uint
+	FishIncrementalInputDecay float64
+	Mean                      float64
+	Varience                  float64
```

Comment:
Are these values specific to fish hunting?

Reply:
Yes they are for the normal distibution in the fish hunt